### DISCUSSION TITLE
Project Name suggestion for PyPI/Readthedocs upload

### DISCUSSION THREAD

**Discussion by codeperfectplus** [Py-Contributors/support#44] [Ideas]
The project name is still in progress. Please suggest a name. Project name required to upload on Pypi and documentation upload.
The current name is not available I am not getting any good idea about it. as both the library are CLI tools. So

- The Project name should be easy and should resemble the project.
- It should not be available on PyPI and readthedocs.

`1. FileMover `
> CLI tool to arrange your files smartly with filemover`
Issue:- https://github.com/Py-Contributors/FileMover/issues/11

`2. Morse`
> Python CLI tool for Morse code encoding and decoding from text and binary
Issue:- https://github.com/Py-Contributors/Morse/issues/3

@Py-Contributors/python-support 
@Py-Contributors/maintainers-team

**Comment by Akshu-on-github** [comment]
`FilesMover` is available if you want something similar to the original name 🤔

I'd suggest `MorseIt` for the second package

**Reply by codeperfectplus** [reply]
Filesmover and MorseIt sounds cool. Thanks for suggestions.

**Comment by ian-dubaideveloper** [comment]
they are cool names

**Reply by codeperfectplus** [reply]
So Filemover was not available on pypi. So we renamed it to `movens` which is the latin word for move

**Reply by codeperfectplus** [reply]
For the Morse code project it's `morseit` as suggested by @Akshu-on-github
